README.md

```diff
@@ -164,7 +164,7 @@ Used to configure the MTU size for attached ENIs. The valid range is from `576`
 
 ---
 
-`AWS_VPC_K8S_CNI_EXTERNALSNAT`
+`AWS_VPC_K8S_CNI_EXTERNALSNAT` (Since v1.6.0)
```

Comment:
This is incorrect I think? I see this envvar existing in branches all the way back to v1.3:

https://github.com/aws/amazon-vpc-cni-k8s/commit/2dd374ef2f7b3284988cc6b68c35418f340af2fb

Reply:
Ugh, thanks, that should be on `AWS_VPC_K8S_CNI_EXCLUDE_SNAT_CIDRS`...